docs/ml-features.md

```diff
@@ -288,6 +288,82 @@ for words_label in wordsDataFrame.select("words", "label").take(3):
 </div>
 
 
+## $n$-gram
+
+An [n-gram](https://en.wikipedia.org/wiki/N-gram) is a sequence of $n$ tokens (typically words) for some integer $n$. The [NGram](api/scala/index.html#org.apache.spark.ml.feature.NGram) class can be used to transform input features into $n$-grams.
```

Comment:
I've started to put the link for the API docs under each language's codetab.  See StringIndexer's doc in this file for an example.